src/components/footer/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./footer.css"
+
+const Footer = ()=>{
+    return(
+        <>
```

Comment:
can remove the <>,  </> since all elements are wrapped inside the <footer>tags

Reply:
ok